examples/svg-interoperability/webpack.config.js

```diff
@@ -15,11 +15,6 @@ module.exports = {
       options: {
         objectAssign: 'Object.assign'
       }
-    }, {
-      // Needed to inline deck.gl GLSL shaders
-      include: [resolve(__dirname, '../../src')],
-      loader: 'transform-loader',
-      options: 'brfs-babel'
```

Comment:
Nit: I still don't see the last line in this example that imports the webpack.local.config, like other examples do. Fine if intentional. Just want to make sure the mechanism is understood.